CONTRIBUTING.md

```diff
@@ -28,25 +28,25 @@ To set up a working **development environment**, ensure that you have `rustup` i
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 git clone git@github.com:[YOUR_FORK_HERE]/surrealdb.git
 cd surrealdb
-cargo run -- -vvv
+cargo run
```

Comment:
The `-vvv` was changed to `--log trace`. For development it's probably better to ensure all logs are written to the output. Could we add the `--log trace` to all of the commands here and below?

`cargo run -- --log trace`

Reply:
Oh, I see. Sure, thanks!

Reply:
Done! I changed this particular one to `cargo run -- help` because `cargo run` prints the help message anyway. This will make it more explicit.